src/js/integrations/reactnativeerrorhandlers.ts

```diff
@@ -58,15 +58,28 @@ export class ReactNativeErrorHandlers implements Integration {
     if (this._options.onunhandledrejection) {
       /*
         In newer RN versions >=0.63, the global promise is not the same reference as the one imported from the promise library.
-        This is due to a version mismatch between promise versions. The version will need to be fixed with a package resolution.
-        We first run a check and show a warning if needed.
+        This is due to a version mismatch between promise versions.
       */
-      this._checkPromiseVersion();
+      this._polyfillPromise();
+    }
+  }
+  /**
+   * Polyfill the global promise instance with one we can be sure that we can attach the tracking to.
+   */
+  private _polyfillPromise(): void {
```

Comment:
Should we use `require.call(null, moduleName).default` instead?
Similar to https://github.com/getsentry/sentry-react-native/commit/600dcb358b54e413f8b952d4d14063f2bc943834

Reply:
We just need to be sure that it works on previous versions of RN.

Reply:
@marandaneto No we don't as 
- `react-native/Libraries/Utilities/PolyfillFunctions` is available [since 0.55](https://github.com/facebook/react-native/commit/f7f5dc66493ad25a85927a9503728b0491c8aab9#diff-82642ae61cae901638245d169cbe2590ff3ecb02e393cba41f07ab0208d475df)
- `promise/setimmediate/**` will always be available as the promise library has been a dependency of RN since [before 0.56.0](https://github.com/facebook/react-native/blob/3c6a916536da67a4591f71883723c67a0c7be9d7/package.json#L185) (our minimum required RN version for this SDK)


Thanks for the double check.

Reply:
Also the require.call hack seemed to have unusual behavior with release/debug builds since last time so we should try to avoid it in future.